app/package.json

```diff
@@ -24,12 +21,11 @@
   "dependencies": {
     "@trussworks/react-uswds": "^6.0.0",
     "@uswds/uswds": "3.7.0",
-    "i18next": "^23.0.0",
+    "lodash": "^4.17.21",
```

Comment:
did we add lodash w/ this PR or is the diff just weird? 🤔

Reply:
I added it in this PR. Its `merge` method is used for merging the default locale to provide fallbacks for missing content in other locales. Its `pick` method would also be eventually used for picking specific content to send into a client-side component (a follow up App router migration PR will demonstrate that).